### PR TITLE
Drop 3.3 support

### PR DESCRIPTION
**Pull request by srittau**
* Merge stdlib/3.3 and ../3.4 into ../3
* Remove conditional branches for Python <3.4

### CODE REVIEW

**Comment by srittau** [comment]
I don't know why pytest is failing. The only change to asyncio/tasks.py was to move it from stdlib/3.4 to stdlib/3.

File: stdlib/3/os/__init__.pyi [hunk 1]
```diff
@@ -17,75 +18,72 @@ _T = TypeVar('_T')
 
 # ----- os variables -----
 
-if sys.version_info >= (3, 2):
```
Comment (JelleZijlstra) [inline stdlib/3/os/__init__.pyi]: I would actually like to keep these in order to make it possible to merge stdlib/2/ and stdlib/3/ in the future.

Comment (srittau) [reply stdlib/3/os/__init__.pyi]: Only in os or in other modules as well?

Comment (JelleZijlstra) [reply stdlib/3/os/__init__.pyi]: Only os.

**Comment by gvanrossum** [comment]
This is a very large diff. @JelleZijlstra, are you comfortable reviewing
the whole thing in one fell swoop?

**Comment by srittau** [comment]
For reference, except for the moved files, the changes just remove version checks and alternative branches, and deindent where necessary. That said, I can easily split this into multiple PRs if desired.

**Comment by gvanrossum** [comment]
I worry about a reviewer glazing over so many similar changes and missing a
tiny but important detail. But it's up to Jelle.

**Comment by srittau** [comment]
Actually, splitting this could be a good idea to localize the pytest failure.

**Comment by gvanrossum** [comment]
OK, go ahead.

**Comment by srittau** [comment]
This is now split into #2262, #2263, #2264, #2265, and #2266. shutil is handled in #2259. asyncio and ipaddress can only be adjusted after #2262 is applied.

**Pull request closed without merging**